src/wacks4shop/wack.py

```diff
@@ -82,9 +85,16 @@ def main(db: Wackabase, dry=False):
 
     args = parser.parse_args()
 
-    # create the database folder if it doesn't exit
-    Path(DATABASE_DIR).mkdir(parents=True, exist_ok=True)
-
-    wackabase = Wackabase(DATABASE_DIR)
-    main(db=wackabase, dry=args.dry)
-    wackabase.write_success()
+    # grab the lock
+    try:
+        lock = FileLock(LOCKFILE, timeout=5)
+        with lock:
+            sleep(30)
+            # create the database folder if it doesn't exit
+            Path(DATABASE_DIR).mkdir(parents=True, exist_ok=True)
+
+            wackabase = Wackabase(DATABASE_DIR)
+            main(db=wackabase, dry=args.dry)
+            wackabase.write_success()
+    except Timeout:
+        logger.info("Lock is acquired! Exiting.")
```

Comment:
shouldn't this message be "couldn't get lock, exiting"

Reply:
ah yeah I meant it in the sense of "lock is already acquired" but I can see how that's confusing!